assets/js/components/chess-board-square.js

```diff
@@ -15,28 +15,41 @@ class ChessBoardSquare extends React.Component {
   }
 
   selectSquare() {
-    var { piece, store } = this.props;
-    var { gameId, selectedSquare } = store.getState();
+    let { piece, store } = this.props;
+    let { gameId, selectedSquare, player } = store.getState();
```

Comment:
Any reason these can't be `const`s instead? Doesn't look like they are modified.